// connect-auth/examples/keys_file.js
exports.bitbucketConsumerKey= "";
exports.bitbucketConsumerSecret= "";
exports.bitbucketCallbackAddress= "http://localhost/auth/bitbucket_callback";
exports.fbId= "238563556250872";
exports.fbSecret= "cfc48e3978736486bd406fbb1c12bad5";
exports.fbCallbackAddress= "localhost/auth/facebook_callback";
exports.getGlueId= "";
exports.getGlueSecret= "";
exports.getGlueCallbackAddress= "http://localhost/auth/getglue_callback";
exports.ghId= "";
exports.ghSecret= "";
exports.ghCallbackAddress= "http://localhost/auth/github_callback";
exports.twitterConsumerKey= "";
exports.twitterConsumerSecret= "";
exports.yahooConsumerKey= "";
exports.yahooConsumerSecret= "";
exports.yahooCallbackAddress= "http://localhost/auth/yahoo_callback";
exports.googleConsumerKey= "";
exports.googleConsumerSecret= "";
exports.googleCallbackAddress= "http://localhost/auth/google_callback";
exports.google2Id= "";
exports.google2Secret= "";
exports.google2CallbackAddress= "http://localhost/oauth2callback";
exports.foursquareId= "";
exports.foursquareSecret= "";
exports.foursquareCallbackAddress= "http://localhost/auth/foursquare_callback";
exports.janrainApiKey= "";
exports.janrainAppDomain= "yourrpxnowsubdomain";
exports.janrainCallbackUrl= "http://localhost/auth/janrain_callback";
exports.openIdCallback= "http://localhost/verify";
exports.linkedinConsumerKey= "";
exports.linkedinConsumerSecret= "";
exports.linkedinCallback= "http://localhost/auth/linkedin_callback";
exports.yammerConsumerKey= "";
exports.yammerConsumerSecret= "";
exports.yammerCallback= "http://localhost/auth/yammer_callback";
